### PR TITLE
Project Finalization

### PR DESCRIPTION
**Pull request by tramstheman**
I have not had the chance to test this much, however the project is now complete given this all works. All we need to do next is put our own "flair" on it I guess to fulfill the assignment 5 reqs. Let me know if there is anything broken, feel free to pr into this branch to make quick fixes. 

- fulfilled CRUD reqs
- Separated CRUD into their own pages
- Added nav bar with drop down
- added stats table that fulfills req agg function reqs
- updated docs
- broke home page into index

### CODE REVIEW

File: add_actor.php [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+        echo "<input type='submit' name='cancel' value='Cancel Submission'>";
+        echo "</form>";
+        echo "</fieldset>";
+    }
+
+    if (isset($_POST['confirmAndSubmit']))
+    {
+        global $dbCon;
+        $null = "NULL";
+        $name = (!empty($_POST['firstNameSubmit']) || !empty($_POST['lastNameSubmit'])) ?
```
Comment (GEvanscsumb) [inline add_actor.php]: This actually happens line 157 which I changed in #11

File: add_movie.php [hunk 2]
```diff
@@ -0,0 +1,283 @@
...
+        $actor3 = (!empty($_POST['actor3_submit'])) ? $_POST['actor3_submit'] : $null;
+        $genre = (!empty($_POST['genre_submit'])) ? $_POST['genre_submit'] : $null;
+        $movie_title = (!empty($_POST['movie_title_submit'])) ? $_POST['movie_title_submit'] : $null;
+        $imdb_link = (!empty($_POST['imdb_link_submit'])) ? $_POST['imdb_link_submit'] : $null;
+        $year = (!empty($_POST['year_submit'])) ? $_POST['year_submit'] : $null;
+        $country = (!empty($_POST['country_submit'])) ? $_POST['country_submit'] : $null;
+        $num_reviews = (!empty($_POST['num_reviews_submit'])) ? $_POST['num_reviews_submit'] : $null;
+        $imdb_score = (!empty($_POST['imdb_score_submit'])) ? $_POST['imdb_score_submit'] : $null;
+
+        createMovie($movie_title, $director, $actor1, $actor2, $actor3, $duration, $genre, 0, "English", $country,
```
Comment (tramstheman) [inline add_movie.php]: ok, the require may have gotten deleted, this should be a quick fix

File: login.php [hunk 3]
```diff
@@ -26,11 +26,11 @@
 <head>
```
Comment (tramstheman) [inline login.php]: i can PR a fix if no one else gets it by the time I get back

File: add_actor.php (hunk 1 above)
Comment (tramstheman) [inline add_actor.php]: not sure, this was done by @KyleLuoma

File: add_movie.php [hunk 4]
```diff
@@ -0,0 +1,283 @@
...
+                    </td>
+                </tr>
+            </table>
+        </form>
+        <?php
+        if(isset($_POST['submit_movie']))
+        {
+            $query = "SELECT actor FROM ACTORS WHERE actor_id = :actor1 || actor_id = :actor2 || actor_id = :actor3";
+            $statement = $dbCon->prepare($query);
+            $statement->execute(array(":actor1" => $_POST['actor1'], ":actor2" => $_POST['actor2'], ":actor3" => $_POST['actor3']));
```
Comment (tramstheman) [inline add_movie.php]: good catch. @KyleLuoma I am guessing this has to do with your loop in the confirm step?

File: add_actor.php (hunk 1 above)
Comment (KyleLuoma) [inline add_actor.php]: Looks like I need to drop the escaped \" from this line.

File: add_movie.php (hunk 4 above)
Comment (KyleLuoma) [inline add_movie.php]: Most likely, I will take a look tonight.

**Pull request merged**